.github/workflows/storybook-chromatic.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         runs-on: ubuntu-latest
         timeout-minutes: 30
         container:
-            image: mcr.microsoft.com/playwright:v1.32.2 # Same as @storybook/test-runner@0.15.2's dependency
+            image: mcr.microsoft.com/playwright:v1.41.2
```

Comment:
@daibhin I saw that you added the initial comment regarding the version lock to 1.32.2 [here](https://github.com/PostHog/posthog/pull/18793#discussion_r1402507723). The dependency is still locked to the same version via @storybook/test-runner@0.16.0 -> jest-playwright-preset@4.0.0 -> playwright-core@1.32.2. Since the CI run is successful I'm assuming everything works with a newer version, but let me know if there's an issue that you know about and that isn't obvious to me.

Reply:
Don't recall the exact context of the comment but I'd imagine it's safe to upgrade. Most likely my CI wasn't passing without it but I think we've even upgraded Storybook a couple of times since so perhaps they loosened their dependencies somewhat